### PR TITLE
v4.12.2

### PR DESCRIPTION
**Pull request by Convly**

### CODE REVIEW

File: examples/getstarted/package.json [hunk 1]
```diff
@@ -12,17 +12,17 @@
...
-    "@strapi/plugin-documentation": "4.12.1",
-    "@strapi/plugin-graphql": "4.12.1",
-    "@strapi/plugin-i18n": "4.12.1",
-    "@strapi/plugin-sentry": "4.12.1",
-    "@strapi/plugin-users-permissions": "4.12.1",
-    "@strapi/provider-email-mailgun": "4.12.1",
-    "@strapi/provider-upload-aws-s3": "4.12.1",
-    "@strapi/provider-upload-cloudinary": "4.12.1",
-    "@strapi/strapi": "4.12.1",
+    "@strapi/icons": "1.8.2",
```
Comment (innerdvations) [inline examples/getstarted/package.json]: was this an accident?

**Review by innerdvations — APPROVED** [review]
LGTM

File: packages/admin-test-utils/tsconfig.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "tsconfig/base.json",
+  "extends": "tsconfig/client.json",
   "compilerOptions": {
     "outDir": "dist"
```
Comment (joshuaellis) [inline packages/admin-test-utils/tsconfig.json:4]:
```suggestion
    "outDir": "dist",
    "noEmit": false
```

Comment (joshuaellis) [reply packages/admin-test-utils/tsconfig.json:4]: @Convly

**Pull request closed without merging**